src/module/Ui/templates/editor/layout.phtml

```diff
@@ -54,10 +57,7 @@ echo $this->doctype();
 </div>
 
 <?php echo $this->partial('layout/partials/default-modal'); ?>
-<?php echo $this->headScript(); ?>
```

Comment:
`headScript` also got removed completely - which module did that come from?

Reply:
It's a ViewHelper from Zf that was only used by Assetic. We can reinclude it again as soon as we don't need Assetic anymore to "compile" the old script.